src/plugins/1/essentials/getting-started/index.md

```diff
@@ -34,87 +20,28 @@ git clone https://github.com/kuzzleio/kuzzle-core-plugin-boilerplate
 cd kuzzle-core-plugin-boilerplate
 docker-compose -f docker/docker-compose.yml up
 // Kuzzle stack with the plugin is ready
-// Edit the file lib/index.js, 
+// Edit the file lib/index.js,
 // the Kuzzle stack will automaticaly restart to include your modifications
 ```
 
-## Prerequisites
-
-### Location
-
-Plugins are subdirectories that must be put at the following location: `<kuzzle_install_dir>/plugins/enabled`.
+The provided `docker-compose` launches a Kuzzle stack with the `pm2` module, you can:
 
-The recommended way to install a plugin is to put it in `plugins/available`, and then link it to the `plugins/enabled` directory.
+- Enable hot reload by watching the plugin's files.
+- Configure it using the file next to the `docker-compose` inside the docker folder.
 
-### Node.js modules
+The main Plugin class is defined in the `index.js`. You can start edit it adding:
 
-Kuzzle loads plugins as [Node.js modules](https://nodejs.org/dist/latest-v8.x/docs/api/modules.html).
+- [Hooks](../hooks)
```

Comment:
I'm not sure this kind of link will work

Reply:
it works

Reply:
IMHO it will be better if we keep the same syntax for links across documentation (`{{ site_base_path }}/.../..`